src/graph/heterograph_capi.cc

```diff
@@ -404,6 +404,7 @@ DGL_REGISTER_GLOBAL("heterograph_index._CAPI_DGLHeteroVertexSubgraph")
       HeteroGraphRef hg = args[0];
       List<Value> vids = args[1];
       bool relabel_nodes = args[2];
+      (void)relabel_nodes;
```

Comment:
why not remove L406?

Reply:
Removing the param 'relabel_nodes' may have compatibility problem, and maybe the param is not implemented yet, I'll confirm it.

Reply:
In past version, we have a check here to confirm relabel_nodes is set to True. However, now we remove this check and enables the function when relabel_nodes is False. 
So it is OK to remove this param here.

Reply:
OK, the param relabel_nodes has been removed